### PR TITLE
Update documentation and dependencies

### PR DESCRIPTION
**Pull request by jlmakes**
I noticed a few problems in the README, eliminating a rollup error and unexpected token errors.

Can I get some eyes on this @artberri?

I've taken over the `karma-rollup-preprocessor` package, and I wanted to make sure your sample configurations for karma are sensible and up-to-date.

### CODE REVIEW

**Comment by jlmakes** [comment]
Hey @artberri, can I get some eyes on this?

I've taken over the `karma-rollup-preprocessor` package, and I wanted to make sure your sample configurations for karma are sensible and up-to-date.

I can update the CHANGELOG and bump to `1.1.1` too if you'd like.

File: README.md [hunk 1]
```diff
@@ -118,10 +118,10 @@ module.exports = function (config) {
     },
     rollupPreprocessor: {
       plugins: [
+        babel(babelrc()),      
         istanbul({
           exclude: ['test/*.js']
-        }),
-        babel(babelrc())
```
Comment (jlmakes) [inline README.md]: This was done deliberately to close #6:
```shell
08 05 2016 22:47:59.218:ERROR [coverage]:
TypeError: Cannot read property 'text' of undefined
    at PROJECT_ROOT/node_modules/istanbul/lib/report/html.js:241:45
```
But doing so, has caused `Unexpected Token` errors for a few other people.

**Comment by jlmakes** [comment]
The previous test failed due to `Permission denied (publickey)` from GitHub when attempting to clone into a bare repository evidently. I replaced the syntax in `package.json`, and everything is all green.

File: package.json [hunk 2]
```diff
@@ -33,12 +33,13 @@
...
   },
   "dependencies": {
-    "istanbul": "^0.4.2",
+    "istanbul": "git@github.com:jansoren/istanbul.git",
     "rollup-pluginutils": "^1.3.1"
   },
   "devDependencies": {
     "babel-cli": "^6.14.0",
     "babel-preset-es2015": "^6.14.0",
+    "babelrc-rollup": "^3.0.0",
```
Comment (jlmakes) [inline package.json]: Oops!

File: package.json [hunk 3]
```diff
@@ -33,12 +33,13 @@
     "lint": "eslint src/index.js test/*.js"
   },
   "dependencies": {
-    "istanbul": "^0.4.2",
+    "istanbul": "git@github.com:jansoren/istanbul.git",
```
Comment (jlmakes) [inline package.json]: This caused `Permission denied (publickey)` error, and bombed the test. However, everything was fixed by simply changing the syntax:

```js
"dependencies": {
    "istanbul": "jansoren/istanbul",
    "rollup-pluginutils": "^1.3.1"
},
```

**Comment by jlmakes** [comment]
Confirmed positive results with #12 using this PR

**Comment by jlmakes** [comment]
@artberri 👀

File: package.json [hunk 4]
```diff
@@ -33,15 +33,15 @@
     "lint": "eslint src/index.js test/*.js"
   },
   "dependencies": {
-    "istanbul": "^0.4.2",
+    "istanbul": "jansoren/istanbul",
```
Comment (jlmakes) [inline package.json]: This repository no longer exists.

**Comment by artberri** [comment]
Sorry for the late response to this @jlmakes . I had been doing other things during all this time and I had this plugin a bit abandoned.

I will not merge your PR because I've finally been able to integrate the new Istanbul instrumenter in the v2.0.0 version. I hope this helps.

**Comment by jlmakes** [comment]
Shit happens, no worries.

Your Karma integration example is wildly out of date however. I took over [`karma-rollup-preprocessor`](https://github.com/jlmakes/karma-rollup-preprocessor) a few months back—so you should update your sample code.

**Pull request closed without merging**